src/components/application_manager/test/resumption/resume_ctrl_test.cc

```diff
@@ -704,6 +797,23 @@ TEST_F(ResumeCtrlTest, IsApplicationSaved) {
   EXPECT_TRUE(res_ctrl->IsApplicationSaved(policy_app_id, device_id));
 }
 
+TEST_F(ResumeCtrlTest, OnAppRegistrationStart) {
+  const std::string kPolicyAppId = "policy_app_id";
+  const std::string kDeviceId = "device_id";
+
+  EXPECT_CALL(*mock_storage, IsApplicationSaved(kPolicyAppId, kDeviceId))
+      .WillOnce(Return(0));
```

Comment:
@LitvinenkoIra , instead of `Return(0)`  think good to create `const bool is_app_saved = false;` and return this const
 `EXPECT_CALL(*mock_storage, IsApplicationSaved(kPolicyAppId, kDeviceId)).WillOnce(Return(is_app_saved));`